docusaurus.config.js

```diff
@@ -9,8 +9,7 @@ const darkCodeTheme = themes.dracula;
 
 require('dotenv').config();
 
-// the default baseUrl is for production deployment, for dev running specify it via DOCS_BASE_URL environment variable
-const baseUrl = process.env.DOCS_BASE_URL || '/api-docs/';
+const baseUrl = '/api-docs/';
```

Comment:
Why not just `/`?
As I understand it, the baseUrl is an url on which the entire website is hosted.